api4/apitestlib.go

```diff
@@ -62,11 +63,21 @@ func UseTestStore(store store.Store) {
 func setupTestHelper(enterprise bool, updateConfig func(*model.Config)) *TestHelper {
 	testStore.DropAllTables()
 
+	tempWorkspace, err := ioutil.TempDir("", "apptest")
```

Comment:
It turns out neither `app` nor `api4` were really setting up their environments correctly, since we couldn't replace the plugin environment after it already started. Pulling this up to set things up more correctly.

Reply:
i felt like it was overwriting, but never could pinpoint. Great fix!